Adding Git in README and some little change

Zangetsu38 opened the pull request:
Fix double version in Gui
Update to version 0.0.6
![capture d ecran 2015-10-01 02 42 11](https://cloud.githubusercontent.com/assets/5261759/10210353/1bbd7b1c-67e6-11e5-9238-3b22462376b4.png) Screen Build Test
Adding a developer.
Add version for Python.

tambry commented on README.md:
```diff
@@ -23,6 +23,7 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
 * [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Git] (https://github.com/git-for-windows/git/releases) (Select "use Git from the Windows Command Prompt" on Adjusting your PATH environement Screen")
```
This is Github for Windows and works only on Windows. https://git-scm.com/downloads would be the right place for actually downloading Git VCS.

Zangetsu38 replied:
oh yes :)

Bigpet commented:
I think the middle part in "v0.0.0.5-2533-gf677782" (the 2533 in this case) is really useful. It's the number of commits since the last version tag.

Just by having "v0.0.0.5 gf677782" and "v0.0.0.5 g4ac6c23" you couldn't tell which one is newer, but with "v0.0.0.5-2533-gf677782" and "v0.0.0.5-2531-g4ac6c23" you can see which one has more commits (doesn't always indicate that it's newer, but it's a reasonably good approximation).

Zangetsu38 commented:
@Bigpet Yes, I know, I tried to keep the number I found trers useful too, but nothing to do, the "Tag" was still there.
Huh? what normal Git? there is only one, the one with my link, there is no other.

Zangetsu38 commented:
@Bigpet Yes, I know.
If anyone knows how to keep the first number "2533-gf677782" only by removing the tag, I'm interested :)
I did not find anything.

tambry commented on README.md:
```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python](https://www.python.org/downloads/) (add to PATH)
```
Why would you remove the version number for Python, which indicated, what version of Python you need?

Zangetsu38 replied:
@tambry I have already explained.

tambry replied:
@Zangetsu38 What? Both? There have been probably hundreds of releases for Python, that are still available. The requirement states clearly, that Python 2.7 or higher is required. And I don't see where you have already explained.

Zangetsu38 replied:
@tambry I speak of version 2.7 and 3.5.
In the "Pull Request" I explained above.
and logically, people click on the link Python, and thus comes to the site, and it can take one he wants.
that's why I is deleted.

tambry replied:
@Zangetsu38 And that's why we specify 2.7+ aka 2.7 or higher - up to the newest version. If we don't specify that, one could even assume that 2.6 or something would work, because the **minimum** required version is not specified. While with the "or higher", one could assume anything above 2.7 will work, including 3.* versions. I don't see how it would confuse anyone, but rather make them aware to update to a newer version (the newest 2.* or 3.*), if they're on a version, that's older.

Zangetsu38 replied:
hmm I see, I Have change That for this

tambry commented on README.md:
```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python 2.7+ Or 3.5+](https://www.python.org/downloads/) (add to PATH)
```
I don't think the 3.5+ is still good. I have Python 3.4.2 and it also works, so 3.5+ is not required. 3.0+ might be better.

Zangetsu38 replied:
@tambry 
hmm, I dunno, those who clicks the link do not really see the version before so much to say that the it will not download the old version is useless, so it's good like that ^^

tambry replied:
@Zangetsu38 It's about stating the smallest version number that you need to compile. 2.7+ is fine - states that 2.7 and any versions above work. I still see no reason to change this, as **any** version above 2.7 will work (that of course includes the 3.* releases. There's a difference between, what's minimum requirement and what's currently recommended.

Zangetsu38 replied:
@tambry 
Yes, I know it, except that if not indicate the minimum Version 3.4+, people will think it does not work with, like nucleus.
Why we have to be clarified.

tambry replied:
@Zangetsu38 I think anyone who's going to compile RPCS3 is not going to be an absolute moron, that doesn't understand what "at least certain version or a newer one" means. If we state anything above 2.7 will work (2.7+), then that will include the latest release, if not stated otherwise. I don't see a reason to fix something, that's not wrong/broke/or in need of clarification.

Zangetsu38 replied:
@tambry Yes, 2.X and 3.X release are very different, and some do not work, is the case with Nucleus, the 3.x version are incompatible, like Xenia.
2.7+ and it just means from 2.7 to 2.7.1 therefore result etc .., out 3.x have nothing to do with it.

tambry replied:
@Zangetsu38 I guess you can keep it like that, if you want. Xenia states that 2.7 is required and doesn't have + sign to indicate that versions above work. For Nucleus, in README I don't even see anything referring to the Python version required. Still you should change 3.5+ to 3.0+, because versions below 3.5 work, as should the 3.* versions starting from 3.0.

Zangetsu38 replied:
@tambry I do not know the date of 3.0 in 2012, it's an old can still ^^ put 3.4+ gosh go?

tambry replied:
@Zangetsu38 It does not matter, when it was released. If it works, guess what, it works!

Zangetsu38 replied:
@I know, it's finally good nothing is left like that, anyway people will not go to download old version anyway ^^.
And even if he had an old it does not matter, it will be an update is like that :)

Zangetsu38 commented:
@Bigpet it's good, I changed everything for git, I handed as before, I changed what was needed to the problem of duplication.
And I saw, I can not do PR for creating a tag, so it would have you, or some other "Owner" were doing.
So it would create the Tag "v0.0.6" as here : https://github.com/Zangetsu38/rpcs3/releases/tag/v0.0.6
You can write what you want to describe.

Aaahh commented:
Make it 2.7+ (includes 3.*)
Or 2.7+ or 3+

Zangetsu38 commented:
@Aaahh I have already explained, it means 2.7+ not also say nothing to do 3.
it just means that it works from version 2.7 and higher than that of, but 3.4 and above have nothing to do with the 2.X is completely different version.
And I will not put 3.0+, the 3.0 is super old.
3.4+ is perfect.

Aaahh commented:
Yeah I'm quite aware of that, hence I didn't change it when I edited the readme. 
Also it's a requirement, not a recommendation...

It's to limit confusion for the noob, and it ends any doubts someone might have about pyhton 3

Zangetsu38 commented:
@Aaahh 
This is just to be more clear, people say how version 2.70 running, and that the 3.5.0 also works, and that if he had a style version from 3.4.0 it is good.
I see written 2.7+ told me I direct, thin, 3.5 not work, this is the case on many project already, why it's important to clear that it works the 3.4 and higher.

Aaahh commented:
@Zangetsu38  yeah thats what I'm saying, we're all good.

tambry commented on README.md:
```diff
@@ -22,7 +22,8 @@ __Windows__
 * [Visual C++ Redistributable Packages for Visual Studio 2015](http://www.microsoft.com/en-us/download/details.aspx?id=48145)
 * [Cmake 3.1.0+](http://www.cmake.org/download/) (add to PATH)
-* [Python 2.7+](https://www.python.org/downloads/) (add to PATH)
+* [Python 2.7+ Or 3.4+](https://www.python.org/downloads/) (add to PATH)
```
@Zangetsu38 Nitpicking again. I believe it should be 3.0+ instead of 3.4+, since 3.0 should work just fine.

Zangetsu38 replied:
@tambry  yes, I know, but it is not serious, there is no point spoke of 3.0, no one uses it, it is much too old.

tambry replied:
@Zangetsu38 I doesn't matter if it's old or "no one" uses it. (I would bet a huge chunk of Python 3.* installations are below 3.4) Requirements are there to state, what version **works**, not what version we recommend you to use because it's "too old". Python 2.7.0 was first released 5 years ago, so by your logic, we shouldn't list it as compatible and simply assume not list it as compatible (while it very very likely is), because it's too old.

Zangetsu38 replied:
@tambry yes, I understand, I did not think that was as old 2.7, but the problem is that someone has already tested the 3.0?
be it walking on well with.

Nekotekina commented:
Cmake and Python are optional requirements.

Zangetsu38 commented:
@Nekotekina I know, it's just for LLVM

danilaml commented:
If we are to bump rpcs3 version we should also tag commit as such.

Zangetsu38 commented:
@danilaml I know I said a bigpet.

Zangetsu38 commented:
@tambry Now is good ?

tambry commented:
Sure, why not, I guess.

Zangetsu38 commented:
@tambry It should also created the TAG, therefore putting v0.0.6, like this style: https://github.com/Zangetsu38/rpcs3/releases/tag/v0.0.6

tambry commented:
@Zangetsu38 That's a real bad tag, with close to info or even a binary. Currently creating an awesome fancy tag.

Zangetsu38 commented:
@ know, it was just for testing ^^
But just need the version is written like this "v0.0.6".
After you wrote what you want in the comments :)

Zangetsu38 commented:
@tambry Oh yes cool, good look :)

Zangetsu38 commented:
@tambry I have a problem, tags not show in Git describe, show hold tag, and i Have compiled build master
![capture d ecran 2015-10-02 16 43 51](https://cloud.githubusercontent.com/assets/5261759/10249321/f86d525a-6924-11e5-8287-ddf92bf7e0af.png)
The commit is good, but tags is hold :/

tambry commented:
@Zangetsu38 I don't understand what you are trying to say. Please use proper English.

Zangetsu38 commented:
@tambry had, rpcs3 always used the old tag, that's normal?

Bigpet commented:
@Zangetsu38 you need to actually fetch the tags, you don't do that with a normal pull

Zangetsu38 commented:
@Bigpet I have already that :/

Zangetsu38 commented:
@Bigpet and @tambry  I found what no works, 
Tags creates the bad commit is made in, I redid a tag, and it works perfectly, the targeted commit is "https://github.com/Zangetsu38/rpcs3/commit/09b894a0f4d92341c80489a6093c0c4c90b663e2"
and not "https://github.com/RPCS3/rpcs3/commit/e273e1719386af457a087399cd6281eeeab355d3"

Bigpet commented:
nope, tagging the merge-commit is fine.

Zangetsu38 commented:
@Bigpet No, I just check everything, it does not work, the tag does not look good commit.
this is done because of that I remade several times the same commit, to market the tag must commit looked good.
and good commit, this is not the last, this is the first one I push.

Bigpet commented:
@Zangetsu38  I don't understand what you're saying, but the tag is fine as it is.

Zangetsu38 commented:
@Bigpet It's simple, the tag is yes, but it does not work to build, it still displays the old, build your own does too.

Bigpet commented:
@Zangetsu38 the build in my repo was made before the tag even existed

Zangetsu38 commented:
@Bigpet hmm ok I do not understand why it works not at home, I'm there deleted, re fork, I still displays the old tag
the tag is displayed on tortoise git, git or tag but spanking git describe, it displays the old, weird.

Zangetsu38 commented:
@Bigpet well, I understand anything, I have everything redone, the market tags, and it points to the same commit as the master ... except the master, its not working, I'm lost.
https://github.com/Zangetsu38/rpcs3/releases/tag/v0.0.6

Zangetsu38 commented:
@Bigpet  ow now It works :)

Bigpet commented:
the tag was not annotated (probably because github created an un-annotated tag when creating a "pre-release")

Zangetsu38 commented:
@Bigpet au i see ^^
